src/components/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <>
```

Comment:
- [ ] While this ensures that this component a single element, the requirements is to wrap it in a `div` tag.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/singleDiv.png)

Reply:
Thank you. It is stated on the pathwright that using `React.Fragment` is a better way to do this other than using `<div>`. I'm confused because we have two different instruction pointing to the same thing.

![22-41-37](https://user-images.githubusercontent.com/26856491/96790378-9c476180-13ee-11eb-81c3-a7cd3ec0f9f2.png)

Note: the above image is from the project specification page on the pathwright page, and [here's](https://reactjs.org/docs/rendering-elements.html) the link provided.

Reply:
@teekaytech don't be confused, there is always an update on the pathwright for better coding practice, perhaps @mosaaleb didn't get to see the update.